### PR TITLE
COL-2168 Modified SQL to only return visible categories in the advanc…

### PR DESCRIPTION
**Pull request by mlincoln**
…ed search
https://jira-secure.berkeley.edu/browse/COL-2168

### CODE REVIEW

File: squiggy/models/course.py [hunk 1]
```diff
@@ -140,6 +140,7 @@ def get_advanced_asset_search_options(cls, course_id):
             SELECT id, title
             FROM categories
             WHERE course_id = :course_id
+            AND visible = True
```
Comment (johncrossman) [inline squiggy/models/course.py]: @mlincoln - Fyi, I typically write the above as `AND visible IS TRUE` but I believe it'll work as is.

Comment (mlincoln) [reply squiggy/models/course.py]: Amended

**Comment by johncrossman** [comment]
Looks good. Waiting on Travis.

File: squiggy/models/course.py [hunk 2]
```diff
@@ -114,8 +114,8 @@ def get_advanced_asset_search_options(cls, course_id):
             'categories': [],
         }
         sql = """
-            SELECT id, CONCAT(category_name, ' - ', name)
-            AS label FROM course_groups
+            SELECT id, CONCAT(category_name, ' - ', name) AS label 
+            FROM course_groups
             WHERE course_id = :course_id
             ORDER BY category_name, name
```
Comment (johncrossman) [inline squiggy/models/course.py:120]: @mlincoln - I think you have trailing whitespace here. The linter (python) is complaining.

**Pull request merged**